### PR TITLE
[BA-793] Implementation of Toast UX Designs

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implemented UX Designs for toast
Fixed Table settings reset button being visible on dashboard tables: https://cofinity-x.atlassian.net/browse/BA-794

### CODE REVIEW

File: cfx-frontend/src/assets/locales/de/common.json [hunk 1]
```diff
@@ -239,17 +244,29 @@
       "declineTitle": "Untersuchung ablehnen",
       "declineReasonHint": "Geben Sie den Grund für die Ablehnung dieser Untersuchung ein",
       "successfullyAccepted": "Untersuchung wurde erfolgreich angenommen.",
+      "successfullyAcceptedTitle": "Untersuchung erfolgreich akzeptieren",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:247]:
```suggestion
      "successfullyAcceptedTitle": "Untersuchung akzeptieren",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 2]
```diff
@@ -239,17 +244,29 @@
...
+      "failedAcceptTitle": "Akzeptieren Untersuchung gescheitert",
       "failedAcknowledge": "Untersuchung konnte nicht bestätigt werden, bitte versuchen Sie es erneut.",
+      "failedAcknowledgeTitle": "Untersuchung als gescheitert bestätigen",
       "failedApprove": "Die Untersuchung konnte nicht genehmigt werden, bitte versuchen Sie es erneut.",
+      "failedApproveTitle": "Genehmigung der Untersuchung gescheitert",
       "failedCancel": "Die Untersuchung konnte nicht gelöscht werden, bitte versuchen Sie es erneut.",
+      "failedCancelTitle": "Untersuchung abbrechen gescheitert",
       "failedClose": "Die Untersuchung konnte nicht abgeschlossen werden, bitte versuchen Sie es erneut.",
-      "failedDecline": "Untersuchung konnte nicht abgelehnt werden, bitte versuchen Sie es erneut."
+      "failedCloseTitle": "Enge Untersuchung fehlgeschlagen",
```
Comment (felixschmidtcofinity) [inline cfx-frontend/src/assets/locales/de/common.json:267]:
```suggestion
      "failedCloseTitle": "Schließen der Untersuchung fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 3]
```diff
@@ -286,17 +303,29 @@
...
       "successfullyCanceled": "Qualitätswarnung wurde erfolgreich abgebrochen.",
+      "successfullyCanceledTitle": "Warnung erfolgreich abbrechen",
       "successfullyClosed": "Qualitätswarnung wurde erfolgreich geschlossen.",
+      "successfullyClosedTitle": "Warnung schließen erfolgreich",
       "successfullyDeclined": "Qualitätswarnung wurde erfolgreich abgelehnt.",
+      "successfullyDeclinedTitle": "Warnung ablehnen erfolgreich",
       "failedAccept": "Qualitätswarnung konnte nicht akzeptiert werden, bitte versuchen Sie es erneut.",
+      "failedAcceptTitle": "Warnung akzeptieren fehlgeschlagen",
       "failedAcknowledge": "Qualitätswarnung konnte nicht bestätigt werden, bitte versuchen Sie es erneut.",
+      "failedAcknowledgeTitle": "Quittierung der Warnung fehlgeschlagen",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:320]:
```suggestion
      "failedAcknowledgeTitle": "Bestätigen fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 4]
```diff
@@ -286,17 +303,29 @@
...
       "failedApprove": "Die Qualitätswarnung konnte nicht genehmigt werden, bitte versuchen Sie es erneut.",
+      "failedApproveTitle": "Warnung genehmigen fehlgeschlagen",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:322]:
```suggestion
      "failedApproveTitle": "Genehmigen fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 5]
```diff
@@ -286,17 +303,29 @@
...
       "failedCancel": "Die Qualitätswarnung konnte nicht gelöscht werden, bitte versuchen Sie es erneut.",
+      "failedCanceTitle": "Warnung abbrechen fehlgeschlagen",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:324]:
```suggestion
      "failedCanceTitle": "Abbrechen fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 6]
```diff
@@ -286,17 +303,29 @@
...
       "failedClose": "Die Qualitätswarnung konnte nicht abgeschlossen werden, bitte versuchen Sie es erneut.",
-      "failedDecline": "Qualitätswarnung konnte nicht abgelehnt werden, bitte versuchen Sie es erneut."
+      "failedCloseTitle": "Warnung schließen fehlgeschlagen",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:326]:
```suggestion
      "failedCloseTitle": "Schließen fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 7]
```diff
@@ -286,17 +303,29 @@
...
+      "failedDecline": "Qualitätswarnung konnte nicht abgelehnt werden, bitte versuchen Sie es erneut.",
+      "failedDeclineTitle": "Ablehnende Warnung fehlgeschlagen"
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:328]:
```suggestion
      "failedDeclineTitle": "Ablehnung fehlgeschlagen"
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 8]
```diff
@@ -239,17 +244,29 @@
...
       "successfullyClosed": "Untersuchung wurde erfolgreich geschlossen.",
+      "successfullyClosedTitle": "Enge Untersuchung erfolgreich",
       "successfullyDeclined": "Untersuchung wurde erfolgreich abgelehnt.",
+      "successfullyDeclinedTitle": "Ablehnende Untersuchung erfolgreich",
       "failedAccept": "Untersuchung konnte nicht akzeptiert werden, bitte versuchen Sie es erneut.",
+      "failedAcceptTitle": "Akzeptieren Untersuchung gescheitert",
       "failedAcknowledge": "Untersuchung konnte nicht bestätigt werden, bitte versuchen Sie es erneut.",
+      "failedAcknowledgeTitle": "Untersuchung als gescheitert bestätigen",
       "failedApprove": "Die Untersuchung konnte nicht genehmigt werden, bitte versuchen Sie es erneut.",
+      "failedApproveTitle": "Genehmigung der Untersuchung gescheitert",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:263]:
```suggestion
      "failedApproveTitle": "Genehmigung der Untersuchung fehlgeschlagen",
```

File: cfx-frontend/src/assets/locales/de/common.json [hunk 9]
```diff
@@ -239,17 +244,29 @@
...
       "failedCancel": "Die Untersuchung konnte nicht gelöscht werden, bitte versuchen Sie es erneut.",
+      "failedCancelTitle": "Untersuchung abbrechen gescheitert",
```
Comment (rogocof) [inline cfx-frontend/src/assets/locales/de/common.json:265]:
```suggestion
      "failedCancelTitle": "Untersuchung abbrechen fehlgeschlagen",
```

**Review by rogocof — CHANGES_REQUESTED** [review]

**Comment by viannbreslerAccso** [comment]
@rogocof Thanks, I made all the required changes

**Review by rogocof — APPROVED** [review]

**Pull request merged**